app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    if @post.save
+      redirect_to post_path(@post), notice: 'Post was successfully created.'
+    else
+      render :new
+    end
+  end
```

Comment:
- Good job done so far, according to the project requirements, your `post` controller should have an `index` action that handles requests for the corresponding route. Please fix this 👍 👍